src/main/resources/changesets/changelog-consent-2024-04-02-rm-old-dbgap-url.xml

```diff
@@ -0,0 +1,18 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet id="changelog-consent-2024-03-28-dataset-dac-id-props.xml" author="grushton">
+    <sql>
+      -- Remove duplicate properties
+      DELETE
+      FROM dataset_property
+      WHERE property_key IN (SELECT key_id FROM dictionary d WHERE KEY = 'dbGap')
+    </sql>
+    <sql>
+      -- Remove deprecated property key from dictionary
+      DELETE
+      FROM dictionary
+      WHERE key = 'dbGap';
```

Comment:
Do we have datasets that ONLY have dbgap and not url? If so, we should also ensure these are converted over.

Reply:
I'll also add that we are not using this value anywhere in the UI any longer either - it's essentially dead data.